apps/gitness/src/pages/pull-request/utils.ts

```diff
@@ -187,6 +187,28 @@ export function extractSpecificViolations(violationsData: any, rule: string) {
   return specificViolations
 }
 
+export const getCommentsInfoData = ({
+  requiresCommentApproval,
+  resolvedCommentArrParams
+}: {
+  requiresCommentApproval: boolean
+  resolvedCommentArrParams?: number[]
+}) => {
+  const resolvedComments = requiresCommentApproval && !resolvedCommentArrParams
+
+  if (resolvedComments) {
+    return { header: 'All comments are resolved', content: undefined, status: 'success' }
+  }
+
+  const unresolvedCount = resolvedCommentArrParams?.length || 0 // Ensure a default value
```

Comment:
shouldn't be length since its a number array should be should something like resolvedCommentArrparams[0]

since object is like 
[2] for resolvedCommentArrParams so right now it'l only ever return 1 if true